include/mxnet/c_api.h

```diff
@@ -1502,22 +1502,23 @@ MXNET_DLL int MXSymbolInferType(SymbolHandle sym,
                                 int *complete);
 
 /*!
- * \brief Convert a symbol into a quantized symbol where FP32 operators are replaced with INT8
- * \param sym_handle symbol to be converted
- * \param ret_sym_handle quantized symbol result
- * \param num_excluded_symbols number of layers excluded from being quantized in the input symbol
- * \param excluded_symbols array of symbols to be excluded from being quantized
- * \param num_offline number of parameters that are quantized offline
- * \param offline_params array of c strings representing the names of params quantized offline
- * \param quantized_dtype the quantized destination type for input data.
- */
-MXNET_DLL int MXQuantizeSymbol(SymbolHandle sym_handle,
-                               SymbolHandle *ret_sym_handle,
-                               const mx_uint num_excluded_symbols,
-                               const SymbolHandle *excluded_symbols,
-                               const mx_uint num_offline,
-                               const char **offline_params,
-                               const char *quantized_dtype);
+* \brief Convert a symbol into a quantized symbol where FP32 operators are replaced with INT8
+* \param sym_handle symbol to be converted
+* \param ret_sym_handle quantized symbol result
+* \param num_excluded_symbols number of layers excluded from being quantized in the input symbol
+* \param excluded_symbols op names to be excluded from being quantized
+* \param num_offline number of parameters that are quantized offline
+* \param offline_params array of c strings representing the names of params quantized offline
+* \param quantized_dtype the quantized destination type for input data.
+* \param disable_requantize whether disable requantize OP during quantization
+* \param calib_quantize whether calibrate quantize op with offline calibration data.
+*/
+int MXQuantizeSymbol(SymbolHandle sym_handle, SymbolHandle *ret_sym_handle,
+                     const mx_uint num_excluded_symbols,
+                     const char **excluded_symbols,
+                     const mx_uint num_offline, const char **offline_params,
+                     const char *quantized_dtype, const bool disable_requantize,
+                     const bool calib_quantize);
```

Comment:
How about wrapping all parameters by a structure so that the API interface doesn't need to change in the future?

Reply:
It's hard to use structure here as it's interface between python and C. Python don't know c structure alignment, it's hard to pack structure from python.